packages/vm/test/tester/index.ts

```diff
@@ -263,6 +263,9 @@ async function runTests() {
         t.ok(assertCount >= expectedTests, `expected ${expectedTests} checks, got ${assertCount}`)
       }
 
+      console.log()
+      console.timeEnd('Total (including setup)')
```

Comment:
Do we want to keep the profile timers here?

Reply:
Yes, I would really like to keep. I need this information repeatedly and it's actually not so easy to place this in "on the fly".